Component/Button/button.css

```diff
@@ -0,0 +1,119 @@
+@import url(\css\nav.css);
+@import url(\css\doc.css);
+@import url(\css\main.css);
+.btn-container {
+    display: flex;
+    background-color: var(--box-color);
+    width: var(--container-width);
+    gap: var(--container-gap);
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+    padding: var(--container-padding);
+}
+
+.btn {
+    margin: 10px;
```

Comment:
It might be better to turn the remaining places with px to rems for consistency

Reply:
Only in places where values are less than 10, there I have used px. At other places I have used rem only.